itests/api/client.go

```diff
@@ -0,0 +1,108 @@
+package api
```

Comment:
Why separate client? May be add those functions to common client in `pkg/client`?